ios/RCTWeChat.m

```diff
@@ -38,13 +37,17 @@ - (void)dealloc
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
-- (BOOL)handleOpenUrl:(NSURL *)aUrl
+- (BOOL)handleOpenURL:(NSNotification *)aNotification
```

Comment:
please refer the document here: https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSNotificationCenter_Class/

Objects register with a notification center to receive notifications (NSNotification objects) using the addObserver:selector:name:object: or addObserverForName:object:queue:usingBlock: methods. Each invocation of this method specifies a set of notifications. Therefore, objects may register as observers of different notification sets by calling these methods several times.

if the the method "handleOpenUrl" use NSURL as the first argument, It will crash immediately whenever one notification is fired.

It should be a NSNotification object.